dev/sensor/bh1750/export/dev/sensor/bh1750.hpp

```diff
@@ -0,0 +1,239 @@
+//!
+//! \file
+//! \brief BH1750 Digital 16bit Serial Output Type Ambient Light Sensor IC
+//!  See, https://e-radionica.com/productdata/BH1750FVI.pdf
+//!
+
+#ifndef __DEV_SENSOR_BH1750_HPP__
+#define __DEV_SENSOR_BH1750_HPP__
```

Comment:
Let's remove `__` from define name. Such names are reserved to the compiler.
>In addition to the names documented in this manual, reserved names include all external identifiers (global functions and variables) that begin with an underscore (‘_’) and all identifiers regardless of use that begin with either two underscores or an underscore followed by a capital letter are reserved names. This is so that the library and header files can define functions, variables, and macros for internal purposes without risk of conflict with names in user programs. 

http://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html